docs/pages/example_usage.rst

```diff
@@ -347,6 +347,12 @@ advanced search
         epsg=32648,
     )
 
+    # use ownership filters
```

Comment:
nice example - can you please ensure `ownership` is also documented in this table: https://capella-console-client.readthedocs.io/en/main/pages/example_usage.html#search-fields

thanks!

Reply:
Definitely!